packages/core/src/StrykerCli.ts

```diff
@@ -1,22 +1,44 @@
-import * as program from 'commander';
+import { Command } from 'commander';
 import { getLogger } from 'log4js';
 import { CONFIG_SYNTAX_HELP } from './config/ConfigReader';
 import { initializerFactory } from './initializer';
 import LogConfigurator from './logging/LogConfigurator';
 import Stryker from './Stryker';
+import { DashboardOptions, StrykerOptions, ALL_REPORT_TYPES } from '@stryker-mutator/api/core';
+import { Config } from '@stryker-mutator/api/config';
+import { Logger } from '@stryker-mutator/api/logging';
+
+function deepOption<T extends string, R>(object: { [K in T]?: R }, key: T, valueInterpreter = (value: R) => value) {
```

Comment:
Could you add comments to explain what this does? Also, I can't find a reference as to a custom `valueInterpreter` argument

Reply:
👍